serverless.yml

```diff
@@ -150,6 +150,9 @@ stepFunctions:
                   ResultPath: $.transformResult
                   TimeoutSeconds: 75
                   Catch:
+                    - ErrorEquals:
+                      - java.lang.RuntimeException
```

Comment:
Not sure if this the best error to key into, but it is the one that's thrown.

Reply:
Let's try to get a more specific error in there RE our discussion about custom exceptions from this morning.

Reply:
Merging this and we'll adjust this definition when the custom exceptions are in place.